.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,11 @@
-## PR Checklist
-
- - [ ] simple searchable title - `[mds-db] Add PG env var`, `[config] Fix eslint config`
- - [ ] briefly describe the changes in this PR
- - [ ] mark as draft if should not be merged
- - [ ] write tests for all new functionality
+## Purpose
+*[description/approach]*
 
-## Impacts
-- [ ] Provider
-- [ ] Agency
-- [ ] Audit
-- [ ] Policy
-- [ ] Compliance
-- [ ] Daily
-- [ ] Native
-- [ ] Policy Author
+## Resolves:
+- [ ] gitmoji + ticket `🐛 JRA-834 fixes bug in the tubes`
```

Comment:
In a public repo I feel like this doesn't make a ton of sense

Reply:
It could be a reference to a Github issue that is resolved, no?